### ISSUE TITLE
notebook 7

### ISSUE THREAD

**Issue by franknoe** [markovmodel/pyemma_tutorials#76]
double-well example:
-----
- vamp is used without any arguments. That is bad practice because at least the user will want to select a lagtime. 
- For this example the default lag time (10) seems too large, because the second singular value only appears to have a contribution of 0.08.
- There is no explanation about the VAMP values, e.g. why the VAMP score is 1 for the bad direction and what this means.

alanine dipeptide
-----
- the RMSD is really pointless, remove that. Use something else instead, such as Cartesian coordinates after aligning to a reference structure. That should still be worse than using the dihedrals.
- I don't think a direct application of VAMP to dihedral makes sense, because dihedrals are periodic but VAMP is linear. Use cos/sin of dihedrals.
- RMSD and dihedral VAMP scores and MSM analyses are in a confusing order, this is hard to follow, needs to be cleaned up.

**I suggest this notebook to be reviewed by @fabian-paul **

**Comment by fabian-paul** [comment]
Hi all, 
I'm happy to review the tutorial. I also want to remind you, that there is already some material for feature selection with VAMP, see https://github.com/markovmodel/pyemma-workshop/blob/master/notebooks/VAMP_01_feature_selection.ipynb 

We could probably also reuse some martial from Frank's slides from the last workshop.

**Comment by marscher** [comment]
On 02.07.2018 21:54, fabian-paul wrote:
> Hi all, 
> I'm happy to review the tutorial. I also want to remind you, that there is already some material for feature selection with VAMP, see https://github.com/markovmodel/pyemma-workshop/blob/master/notebooks/VAMP_01_feature_selection.ipynb 
> 
> We could probably also reuse some martial from Frank's slides from the last workshop.
> 

Hi Fabian,

we've decided that we are going to restructure the tutorial a bit. The
VAMP estimation and scoring will be demonstrated along with a comparison
with TICA and PCA, so please standby with reviewing this right now.
Thank you for pointing us to the the previous tutorial, this is helpful.

Best,
Martin
